13-foundations-mathematical.Rmd

```diff
@@ -25,6 +25,17 @@ While they differ in the settings, in their outcomes, and also in the technique
 terms_chp_13 <- c("sampling distribution", "null distribution")
 ```
 
+::: {.important data-latex=""}
+**Sampling distribution.**
+
+A sampling distribution describes the possible values of a **statistic** (note that here the sample proportion, $\hat{p}$ is described, but other statistics like the sample mean, $\bar{x}$ will be described later) as they change from study to study.\
```

Comment:
```suggestion
A sampling distribution describes the possible values of a **statistic** as they change from sample to sample. Here, the sample proportion ($\hat{p}$) is described, but other statistics like the sample mean ($\bar{x}$) will be described later.
```

Reply:
i'm 100% fine with this, and i do think it is how the term is typically defined.  

but i think there is something more to the idea of a sampling distribution.  for example, if the question is one of randomization, then the sampling distribution represents the possible values over all randomizations.  that's why i chose the words "study" instead of "sample".  (especially given how we've focused on the computational creation via randomization tests).

add a caveat that says randomization sampling distributions are also sampling distributions?  or no, just leave it at sampling?